pkg/pb/health.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package health;
+
+import "gogo.proto";
+import "pointerdb.proto";
+
+service Health {
+    rpc ObjectStat(ObjectHealthRequest) returns (ObjectHealthResponse) {}
```

Comment:
`ObjectHealth`, request and response should match.

Reply:
done